tools/BuildMonitor/BuildMonitor.csproj

```diff
@@ -51,6 +51,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DeployExtension>False</DeployExtension>
```

Comment:
This setting is used to deploy an extension into an experimental instance of VS for debugging.  Setting this to false for regular builds so that we don't hit this error:  VSSDK1031.